test/custom-style-extension/before-fixes.html

```diff
@@ -0,0 +1,32 @@
+<style is="custom-style">
```

Comment:
there's some tricky indentation code in this PR. let's add in another style node that already has some indentation, to ensure it's handling it correctly

Reply:
Good call! Wasn't properly handling that case and realized I left some duplicate code in there.